dataland-frontend/tests/e2e/fixtures/eutaxonomy/financials/EuTaxonomyDataForFinancialsPreparedFixtures.ts

```diff
@@ -173,3 +176,51 @@ function createAllValuesCompany(
   };
   return input;
 }
+
+function createCreditInstitutionWithEligibleActivity29(
```

Comment:
These 3 functions seem to be rather redundant. Why not unify them into 1 that gets the changing value as parameter?

Reply:
Will fix this now.

Reply:
Fixed by using a higher-order-function to return the correct generator function based on the desired value of the field "eligibleActivity".
Feel free to re-review this.